app/models/registration.rb

```diff
@@ -245,7 +252,7 @@ def registration_history
         actor_id: r.actor_id,
         timestamp: r.created_at,
         action: r.action,
-      }
+      }.with_indifferent_access
```

Comment:
Not happy about this in our main code. Why do you need it?

Reply:
The hash we were getting from this without intervention is the following: 
```ruby
{changed_attributes: {"auto_accept_failure_reasons" => "-7001"}, actor_type: "System", actor_id: "auto-accept", timestamp: Tue, 06 May 2025 03:19:41.320953000 UTC +00:00, action: "System reject"}
```

This was happening because `changed_attributes` was a hash populated with strings from the database - I'm now calling `symbolize_keys` on it when it's created, which fixes the issue

Reply:
Muuuch better :relieved: